mlflow/metrics/utils/metric_definitions.py

```diff
@@ -43,7 +44,15 @@ def _toxicity_eval_fn(eval_df, metrics):
     toxicity_ratio = toxicity.compute(predictions=predictions, aggregation="ratio")[
         "toxicity_ratio"
     ]
-    return MetricValue(scores=scores, aggregate_results={"ratio": toxicity_ratio})
+    return MetricValue(
+        scores=scores,
+        aggregate_results={
+            "ratio": toxicity_ratio,
+            "mean": np.mean(scores),
+            "variance": np.var(scores),
+            "p90": np.percentile(scores, 90),
```

Comment:
can we pull these out into a helper function like this
```
def standard_aggregations(scores):
    return {
        "mean": np.mean(scores),
        "variance": np.var(scores),
        "p90": np.percentile(scores, 90),
    }
```

then we can do something here like 
```
aggregate_results={
    ...standard_aggregations(scores),
    "ratio": toxicity_ratio,
}
```

and just `aggregate_results=standard_aggregations(scores)` for the other metrics